aten/src/ATen/test/pow_test.cpp

```diff
@@ -35,16 +35,16 @@ const std::vector<int> ints {
   int_min,
   int_min + 1,
   int_min + 2,
-  static_cast<int>(-sqrt(int_max)),
+  static_cast<int>(-sqrt(static_cast<double>(int_max))),
```

Comment:
Might be good to just leave a comment in the PR review section here as to why we're making this change so we have a record of it